src/acrolinx-plugin.ts

```diff
@@ -158,6 +159,9 @@ function initAcrolinxSamplePlugin(config: AcrolinxPluginConfig, editorAdapter: A
       },
 
       onCheckResult(checkResult: CheckResult) {
+        if (checkResult.embedCheckInformation && config.onEmbedCheckData) {
+          config.onEmbedCheckData(checkResult.embedCheckInformation, checkResult.inputFormat||"");
```

Comment:
tslint complains about missing white space around the  `||`.
This can be fixed automatically by running `npm run tslintFix`

@acrork If the future brings more PRs, we should think about running a test build on https://travis-ci.org/ . This way a pull requests would be automatically checked for such simple problems. Do you know if we have already travis jobs? I remember vaguely that @ska2342 talked about it, but there was a kind of problem.

Reply:
@acromarco  Fixed issue reported by tslint at:
ERROR: src/acrolinx-plugin.ts[163, 93]: missing whitespace
ERROR: src/acrolinx-plugin.ts[163, 95]: missing whitespace